### ISSUE TITLE
Handle disabling submitting vote result when voteResult is a tie and voting period done

### ISSUE THREAD

**Issue by jtrein** [openlawteam/molochv3-ui#260]
We should disable the "submit vote result" button when no one has voted.

Depends on: https://github.com/openlawteam/molochv3-contracts/pull/256

**Comment by jdville03** [comment]
As discussed with @adridadou, the changes needed for the UI are the following:
1. If proposal has no votes and the voting period has ended, don't show the "Submit Vote Result" button.
2. Show Grace Period timer immediately after the voting period has ended.
3. Show disabled "Process" button if still in Grace Period.
4. Enable "Process" button when the Grace Period is over.

To summarize, if the proposal has no votes, the "Submit Vote Result" action should be skipped and go right to Grace Period and then ability to Process.

**Comment by jtrein** [comment]
As far as the listing categories are concerned, it's my inclination to keep it in the "Voting" section, until a result is submitted, just like how it's handled today. Thoughts?

**Comment by jdville03** [comment]
In this case there will be no vote result submitted. But I agree that we should keep the category like how it's handled today. It can stay in the "Voting" section until the Grace Period has ended. After that, it should be in the "Failed" section with the Process button re-enabled.

**Comment by jtrein** [comment]
Thank you, you're right. OK, in that case I will need to thread through the Snapshot Hub votes and their results into the `<Proposals />` to be able to determine for certain no one voted. It will fit nicely with my work in #248, though I will make a separate PR.
